app/controllers/webhooks/github_controller.rb

```diff
@@ -39,9 +39,12 @@ def push_event(chat, body)
     end
 
     msg = msg % [body['pusher']['name'], body['commits'].length, body['compare']]
+    body['commits'].each do |commit|
+      msg += "\n%s - %s" % [commit['id'][0..5], commit['message']]
```

Comment:
Why 5? Does it leave last 5 commits or 5 first commits?

Honestly, I don't have strong opinion about how many commits should be displayed or should all of them displayed. But we need to be sure that they are displayed in the same way as 'git log' does it. Also, if we truncate comments, I would add '...' at the end to show that there are more commits.